packages/calendar/stories/calendar.stories.ts

```diff
@@ -62,114 +98,41 @@ const renderCalendar = (
             `.story-container-${randomId}`
         );
 
-        if (container) {
-            render(story, container as HTMLElement);
-        }
+        if (container) render(story, container as HTMLElement);
```

Comment:
Can this be done in a different manner or should we stick to `requestAnimationFrame`?
It seems like without it the component renders only the header (month navigation buttons and the current month indicator) and not the grid of days. Does this hide a problem in the component's lifecycle? 🤔

Reply:
It seems like the issue comes from the interaction of `sp-theme` and `LanguageResolutionController`. Explicitly that the controller's `hostConnected` is called before the `sp-theme`'s constructor, causing the `sp-language-context` event to be dispatched before the event listener for said event is added in `Theme.ts`. 
This looks like a hack-ish workaround for a more general problem that I recreated outside of the storybook as well: https://studio.webcomponents.dev/edit/29O0MXJBQtaMF3CgjhhW/src/index.ts?p=stories. Here you can see that even if the `sp-theme`'s `lang` attribute is changed, the language of the `number-field` coming from the controller didn't change.
More investigation is needed here because I don't yet know how this interaction can be fixed. I would be happy to get any help or suggestions.

Reply:
Two possible solutions that I've found are:
1. Set `bundled` to `false` in the `swcThemeDecoratorWithConfig` ([here](https://github.com/adobe/spectrum-web-components/blob/v0.46.0/projects/story-decorator/decorator.ts#L35)) - I'm not aware of all the implications and where is this variable set other than the default value.
2. Add one more call of `resolveLanguage` in the `LanguageResolutionController` using the `hostUpdate` (making sure to call this only for the first update). For environments other than the storybook this one extra call should not be a problem because `Theme` checks if the `contextConsumer` was already subscribed ([here](https://github.com/adobe/spectrum-web-components/blob/main/tools/theme/src/Theme.ts#L646)). 

Personally I'm leaning more towards the second solution. What do you think about this @Rajdeepc ?

Reply:
Do you need more discussion on this @mizgaionutalexandru . I think adding a `@state()` decorator will reconcile the changes!

Reply:
Yes, I would love to clarify this because there are 2 different topics. The `state()` decorator works for 'normal app environments' let's say, and that's how the storybook decorator is implemented but due to the fact that in **storybook** the story function gets called before the `sp-theme`'s constructor gets the chance to add the event listener that is needed for the language resolver.
I found that those 2 approaches would fix the issue in the storybook environment.